src/samples/EducationStart/index.tsx

```diff
@@ -445,7 +451,9 @@ const EducationStartCase: FunctionComponent<any> = () => {
         value={{
           appBacklinkProps: {},
           showLanguageToggle,
-          serviceParam: educationStartParam
+          serviceParam: educationStartParam,
+          serviceName: serviceNameAndHeader,
```

Comment:
We could keep `serviceName` here and in the header component we could fetch this value and display the correct header.

Reply:
can we connect for this?

cookiePage component is having a separate AppHeader import, its not getting fetched from '/EducationStart/index.